sqlmesh/core/model/kind.py

```diff
@@ -88,6 +88,7 @@ class ModelKindName(str, ModelKindMixin, Enum):
     EMBEDDED = "EMBEDDED"
     SEED = "SEED"
     EXTERNAL = "EXTERNAL"
+    NONE = "NONE"
```

Comment:
Where is this used?

Reply:
Default value for `SnapshotTableInfo` `kind_name`, so that `model_kind_name` doesn't have to be optional (which had a spider effect throughout a lot of code).

Reply:
It did? I thought the impact was minimal since we did `==` everywhere. I could only find like 2 places where it matters. Did you find more?

Reply:
The method originates from the `ModelKindMixin` and making the return type optional there is what caused the issue.